### ISSUE TITLE
AspectToken remove will trigger unrecognised exception if there is other aspect registered.

### ISSUE THREAD

**Issue by yanjunz** [steipete/Aspects#31]
a temporary fix: https://github.com/yanjunz/Aspects/commit/c90cb1bad9a7869f3b0eaccffa47c43985fc830e

**Comment by itsthejb** [comment]
You mean that you will get an `unrecognized selector` exception if you register an aspect for that selector, later remove it, but also have other swizzled selectors remaining?

If so, also see this issue and will try your fix.

**Comment by yanjunz** [comment]
Yes. I have created a test case in the demo project. You can use it to reproduce this issue.
https://github.com/yanjunz/Aspects
